packages/eslint-plugin-template/src/rules/accessibility-valid-aria.ts

```diff
@@ -0,0 +1,60 @@
+const { aria } = require('aria-query');
+
+import {
+  createESLintRule,
+  getTemplateParserServices,
+} from '../utils/create-eslint-rule';
+
+const ariaAttributes = new Set<string>(aria.keys());
+
+type Options = [];
+export type MessageIds = 'accessibilityValidAria';
+export const RULE_NAME = 'accessibility-valid-aria';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Ensures that the correct ARIA attributes are used.',
+      category: 'Best Practices',
+      recommended: false,
+    },
+    schema: [],
+    messages: {
+      accessibilityValidAria:
+        '{{attribute}}: This attribute is an invalid ARIA attribute.',
+    },
+  },
+  defaultOptions: [],
+  create(context) {
+    const parserServices = getTemplateParserServices(context);
+
+    return {
+      // Note that we're using the `Element` visitor except of `TextAttribute`
```

Comment:
You need to update the visitor keys in the template-parser `index.ts`, `attributes` should be added to the existing `Element` definition, and any additional key types should be added there as and when required

Reply:
Thanks, updated.